app/src/engineering/res/xml/network_security_config.xml

```diff
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<network-security-config>
-    <base-config cleartextTrafficPermitted="true">
-        <trust-anchors>
-            <!-- Trust preinstalled CAs -->
-            <certificates src="system" />
-            <!-- HERE: Additionaly trus user added CAs -->
-            <certificates src="user"/>
-        </trust-anchors>
-    </base-config>
-</network-security-config>
```

Comment:
This was used by Martyna to install some network interceptor to log the network. We no longer are using that approach so removing it completely makes sense.